src/quo/components/wallet/token_input/style.cljs

```diff
@@ -12,30 +12,22 @@
   {:padding-horizontal 20
    :padding-bottom     4
    :height             36
-   :flex-direction     :row
-   :justify-content    :space-between})
+   :flex-direction     :row})
 
 (defn token-name
   [theme]
   {:color          (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)
-   :margin-right   8
-   :padding-bottom 2})
+   :padding-bottom 3})
 
-(def token-label-container
-  {:position       :absolute
-   :left           40 ; token image size + margin
-   :right          0
-   :bottom         0
-   :top            0
-   :flex-direction :row
+(def text-input-container
+  {:flex-direction :row
    :align-items    :flex-end})
 
-(def text-input-container
-  {:position :absolute
-   :top      0
-   :bottom   0
-   :left     40 ; token image size + margin
-   :right    0})
+(defn input-container
+  [window-width]
+  {:width        (- window-width 120)
```

Comment:
what is 120 related to?

Reply:
Token icon width + swap button width + spacing.

I will add a comment to make it more clear.